Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './lib/python3.5/site-packages/treatyofbabel'...
Compiling './lib/python3.5/site-packages/treatyofbabel/babel.py'...
***   File "./lib/python3.5/site-packages/treatyofbabel/babel.py", line 62
    raise BabelError, 'Unknown story format'
                    ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/treatyofbabel/formats'...
Compiling './lib/python3.5/site-packages/treatyofbabel/formats/quest.py'...
***   File "./lib/python3.5/site-packages/treatyofbabel/formats/quest.py", line 75
    raise Exception, "No game information found"
                   ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/treatyofbabel/formats/tads2.py'...
***   File "./lib/python3.5/site-packages/treatyofbabel/formats/tads2.py", line 137
    raise Exception, "Image corrupted: cannot determine dimensions"
                   ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/treatyofbabel/formats/tads3.py'...
***   File "./lib/python3.5/site-packages/treatyofbabel/formats/tads3.py", line 137
    raise Exception, "Image corrupted: cannot determine dimensions"
                   ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/treatyofbabel/ifiction.py'...
***   File "./lib/python3.5/site-packages/treatyofbabel/ifiction.py", line 165
    raise IFictionError, "identification: IFID required"
                       ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/treatyofbabel/ifstory.py'...
***   File "./lib/python3.5/site-packages/treatyofbabel/ifstory.py", line 199
    raise Exception, "No IFID or TUID set"
                   ^
SyntaxError: invalid syntax

Listing './lib/python3.5/site-packages/treatyofbabel/utils'...
Listing './lib/python3.5/site-packages/treatyofbabel/wrappers'...
Compiling './lib/python3.5/site-packages/treatyofbabel/wrappers/blorb.py'...
***   File "./lib/python3.5/site-packages/treatyofbabel/wrappers/blorb.py", line 89
    raise Exception, "Image corrupted: cannot determine dimensions"
                   ^
SyntaxError: invalid syntax

Listing './share'...
Listing './share/python-wheels'...
